jekyll/_includes/snippets/troubleshoot/self-hosted-runner-troubleshoot-snip.adoc

```diff
@@ -152,4 +152,4 @@ Cancel the job and rerun it. If your job is still not running, file a link:https
 
 CircleCI's machine runners support rerunning a job with SSH for debugging purposes. Instructions on using this feature can be found at <<ssh-access-jobs#,Debugging with SSH>>.
 
-NOTE: The `Rerun job with SSH` feature is disabled by default. To enable this feature, see xref:runner-config-reference.adoc#runner-ssh-advertise_addr[Installing the CircleCI's Self-hosted Runner]. Rerun job with SSH is not currently available with container runner.
+NOTE: The `Rerun job with SSH` feature is disabled by default. To enable this feature, see xref:runner-config-reference.adoc#runner-ssh-advertise_addr[Installing the CircleCI's Self-hosted Runner]. 
```

Comment:
Is it worth saying 

"To enable this feature, see the xref:runner-config-reference.adoc#runner-ssh-advertise_addr[Machine Runner Instructions] or the xref:container-runner-installatiion.adoc#enable-rerun-job-with-ssh[Container Runner instructions]."